cmd/client-s3-trace_v4.go

```diff
@@ -61,6 +61,12 @@ func (t traceV4) Request(req *http.Request) (err error) {
 
 		// Undo
 		req.Header.Set("Authorization", origAuth)
+	} else {
+		var reqTrace []byte
```

Comment:
Shall we declare `reqTrace` in L40 itself ? after `origAuth := req.Header.Get("Authorization")` ?

Reply:
IMO why to declare the variable if its not used. Since `reqTrace` is only used in `else` condition so it is rightly placed.

Reply:
It is also used in the if clause ( if authorization header is present ). So it can be generically declared

Reply:
rearranged it a little bit. PTAL